packages/site/src/pages/submit.astro

```diff
@@ -312,9 +316,15 @@ Stable diffusion for the win. For images and animations. But what is facebook do
       return;
     }
 
-    const formData = new FormData(form);
-    const data = Object.fromEntries(formData.entries()) as Meetup['data'];
-    checkMeetupData(data);
+    const formData = Object.fromEntries(
+      new FormData(form).entries(),
+    ) as MeetupFormValues;
+    checkMeetupData(formData);
```

Comment:
Voi jättää nytkin näin, mutta mielummin:
- `const formData: unknown = ...`
- `assertFormData(formData)` -> `formData is MeetupFormValues`